app/controllers/sign_in/client_configs_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module SignIn
+  class ClientConfigsController < SignIn::ApplicationController
+    rescue_from ActiveRecord::RecordNotFound, with: :not_found
+    skip_before_action :authenticate
+    before_action :authenticate_service_account
+    before_action :set_client_config, only: %i[show update destroy]
+
+    def index
+      client_configs = if params[:client_ids].presence
+                         SignIn::ClientConfig.where(client_id: params[:client_ids])
+                       else
+                         SignIn::ClientConfig.all
```

Comment:
I don't think we want to ever display all client configs, since that could cross team boundaries. But we might want to display all client configs for a team. I'm not sure if client configs should have a 'team' field, or if identity dashboard should just specifically ask for all the client configs it wants to see

Reply:
Yeah, good call. Identity dashboard will always ask for which ones it needs